src/codecoverage/bot/code_coverage_bot/github.py

```diff
@@ -46,6 +47,13 @@ def mercurial_to_git():
             r = requests.get('{}/gecko-dev/rev/hg/{}'.format(self.hg_git_mapper, mercurial_commit))
 
             if not r.ok:
+                for email in secrets[secrets.EMAIL_ADDRESSES]:
```

Comment:
The people we are going to notify for missing commits are not the same we notify in notifier.py for 0 coverage, so we should have an additional secret for it.
You can call it REPO_MAPPER_EMAIL_ADDRESSES (you will have to add it in secrets.py).